Simple-PHP-IPAM/lib.php

```
@@ -3923,6 +3931,279 @@ function ipv4_assignable_count(int $prefix): int
+
+        $sum = $base;
+        foreach (($children[$id] ?? []) as $cid) {
+            $c = $sumNode((int)$cid);
+            $sum['assignable_total']      += $c['assignable_total'];
+            $sum['assigned_assignable']   += $c['assigned_assignable'];
+            $sum['unassigned_assignable'] += $c['unassigned_assignable'];
+        }
+        return $agg[$id] = $sum;
+    };
+
+    foreach ($tree['byId'] as $id => $_row) $sumNode((int)$id);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$_row'. (undefined)

(UnusedLocalVariable)